src/xbps/repositories/mirrors/index.md

```diff
@@ -29,32 +29,32 @@ sub-repository.
 
 ### Globally-available mirrors
 
-| Repository                                         | Location          |
-|----------------------------------------------------|-------------------|
-| <https://mirror.ps.kz/voidlinux/>                  | Asia: Almaty, KZ  |
-| <https://mirrors.bfsu.edu.cn/voidlinux/>           | Asia: China       |
-| <https://mirrors.cnnic.cn/voidlinux/>              | Asia: China       |
-| <https://mirrors.tuna.tsinghua.edu.cn/voidlinux/>  | Asia: China       |
-| <https://mirror.sjtu.edu.cn/voidlinux/>            | Asia: China       |
-| <https://mirror.nju.edu.cn/voidlinux/>             | Asia: China       |
-| <https://void.webconverger.org/>                   | Asia: Singapore   |
-| <https://mirror.aarnet.edu.au/pub/voidlinux/>      | AU: Canberra      |
-| <https://ftp.swin.edu.au/voidlinux/>               | AU: Melbourne     |
-| <https://voidlinux.com.br/repo/>                   | BR: Ouro Preto    |
-| <https://void.cijber.net/>                         | EU: Amsterdam, NL |
-| <http://ftp.dk.xemacs.org/voidlinux/>              | EU: Denmark       |
-| <https://mirrors.dotsrc.org/voidlinux/>            | EU: Denmark       |
-| <https://quantum-mirror.hu/mirrors/pub/voidlinux/> | EU: Hungary       |
-| <https://voidlinux.mirror.garr.it/>                | EU: Italy         |
-| <http://ftp.debian.ru/mirrors/voidlinux/>          | EU: Russia        |
-| <https://mirror.yandex.ru/mirrors/voidlinux/>      | EU: Russia        |
-| <https://mirror.accum.se/mirror/voidlinux/>        | EU: Sweden        |
-| <https://ftp.lysator.liu.se/pub/voidlinux/>        | EU: Sweden        |
-| <https://void.sakamoto.pl/>                        | EU: Warsaw, PL    |
-| <https://mirror.vofr.net/voidlinux/>               | USA: California   |
-| <https://mirror2.sandyriver.net/pub/voidlinux/>    | USA: Kentucky     |
-| <https://mirror.clarkson.edu/voidlinux/>           | USA: New York     |
-| <https://mirror.puzzle.ch/voidlinux/>              | EU: Bern, CH      |
+| Repository                                         | Region 		       | Location      	 	           |
+|----------------------------------------------------|-----------------|-----------------------------|
+| <https://mirror.ps.kz/voidlinux/>                  | Asia		          | Almaty, Kazakhstan	 	       |
+| <https://mirrors.bfsu.edu.cn/voidlinux/>           | Asia		          | China			 	                  |
+| <https://mirrors.cnnic.cn/voidlinux/>              | Asia		          | China			 	                  |
+| <https://mirrors.tuna.tsinghua.edu.cn/voidlinux/>  | Asia		          | China			 	                  |
```

Comment:
`mirrors.cnnic.cn` is a `CNAME` to `mirrors.tuna.tsinghua.edu.cn`, they're both placed in Beijing